crates/libcontainer/src/process/container_init_process.rs

```diff
@@ -4,14 +4,15 @@ use crate::syscall::Syscall;
 use crate::workload::ExecutorManager;
 use crate::{
     capabilities, hooks, namespaces::Namespaces, process::channel, rootfs::RootFS,
-    rootless::Rootless, seccomp, tty, utils,
+    rootless::Rootless, tty, utils,
 };
 use anyhow::{bail, Context, Ok, Result};
 use nix::mount::MsFlags;
 use nix::sched::CloneFlags;
 use nix::sys::stat::Mode;
 use nix::unistd::setsid;
 
+use log::warn;
```

Comment:
Hey, it seems this is making clippy fail. Can you remove this and check if it fixes it?